data/templates/set_bmc_credentials.sh

```diff
@@ -1,9 +1,23 @@
 #!/usr/bin/env bash
 
+channel=''
+function set_channel()
+{
+    for i in {1..15}; do
+        ipmitool -I lanplus -H 172.31.128.2 -U admin -P admin user list $i &>/dev/null
```

Comment:
Of course hardcoding won't help :) I missed editing this line from my test script. Will fix it.